pkg/api/customization/cluster/formatter.go

```diff
@@ -47,23 +47,21 @@ func (f *Formatter) Formatter(request *types.APIContext, resource *types.RawReso
 			resource.AddAction(request, v3.ClusterActionBackupEtcd)
 			resource.AddAction(request, v3.ClusterActionRestoreFromEtcdBackup)
 		}
-		canUpdateCluster := func(request *types.APIContext, clusterID string) bool {
+		canUpdateClusterFn := func(request *types.APIContext, clusterID string) bool {
```

Comment:
why is this an embedded function?

Reply:
I saw similar code style being used at multiple other places, so I followed the same. For example: https://github.com/rancher/rancher/blob/master/pkg/api/customization/cluster/actionhandler.go#L30

I guess the embedded function improves code readability?